mungegithub/mungers/submit-queue.go

```diff
@@ -866,22 +866,34 @@ func (sq *SubmitQueue) validForMerge(obj *github.MungeObject) bool {
 	}
 
 	// Clearly
-	if !obj.HasLabel(lgtmLabel) {
+	if !(obj.HasLabel(lgtmLabel) || obj.HasLabel(approvedLabel)) {
```

Comment:
In future we would break this down into two separate conditions? Because I imagine we want to set the status differently in each case.